.gitignore

```diff
@@ -21,6 +21,7 @@ docs/source/api/test
 docs/source/api/utils
 docs/source/api/plugins
 docs/source/api/optional-plugins
+docs/source/guides/writer/libs/vmimage.rst.data
```

Comment:
My only suggestion is to rename this dir. maybe `data/vmimage.rst`

Reply:
Yes, you are right, It will be better for future extension.

Reply:
I force-pushed the change here